src/frontend/sider/Fagsak/Behandling/Venstremeny/Venstremeny.tsx

```diff
@@ -95,7 +91,7 @@ const Venstremeny: React.FunctionComponent = () => {
     return (
         <HStack justify="start">
             {åpenVenstremeny && (
-                <Meny as="nav">
+                <Box as="nav" paddingBlock="8">
```

Comment:
Her vet jeg ikke helt hva som er riktig, men burde man brukt `ASpacing8` fremfor bare "8"? 🤔

Reply:
Godt spørsmål! 😊 Disse verdiene mapper om til [spacing tokens](https://aksel.nav.no/grunnleggende/styling/design-tokens#0cc9fb32f213) i Aksel. Så selv om det står 8 her, betyr det egentlig 2rem. ☺️

Reply:
Har gått over til de nye space tokenene for Aksel i formatet `space-x`. ☺️